.github/workflows/on_PR_windows_matrix.yml

```diff
@@ -91,59 +90,59 @@ jobs:
       - name: Test
         if: ${{matrix.platform == 'x64'}}
         run: |
-          ctest --test-dir build --output-on-failure
+          ctest --test-dir build --output-on-failure --stop-on-failure
```

Comment:
Would this stop the testing when it encounters the first failing test?       
I think I prefer if the ci shows me all failing tests, not just the first one.

Reply:
Yes, I knew this could generate some controversy 😇 . I added it because at some point the tests on MSYS were taking more than 1 hour and I wanted to break as soon as possible.

It is fine for me to revert these changes now that everything is working.